CSL JSON.js

```diff
@@ -134,6 +134,7 @@ var testCases = [
 					{
 						"lastName": "Bachar",
 						"firstName": "Mostafa",
+						"creatorTypeID": 1,
```

Comment:
This shouldn't be here — those ids aren't meant to be fixed and it's redundant with `creatorType`. Is this coming through from Scaffold?

Reply:
Yes, from Scaffold, you also see them in the automatic translator tests.